compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangMarkDownDeprecationDocumentation.java

```diff
@@ -53,6 +53,11 @@ public void addDeprecationLine(String text) {
         }
     }
 
+    @Override
+    public String getDocumentation() {
+        return String.join("\n", this.deprecationDocumentationLines).replaceAll("\r", "");
```

Comment:
Any specific reason to replace the `CR` here?

Reply:
The other related nodes have the same implementation. So followed the same here to be consistent. Not entirely sure why it was originally replaced though. Maybe to be consistent?
https://github.com/ballerina-platform/ballerina-lang/blob/65396a2b8b11bd01d66f695978816f9d53e01804/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangMarkdownDocumentation.java#L106